src/Discovery/test/HttpClients.Test/RegisterMultipleDiscoveryClientsTest.cs

```diff
@@ -132,10 +132,10 @@ public async Task WithEurekaClientCertificateConfiguration_AddsDiscoveryClient()
 
         WebApplicationBuilder builder = WebApplication.CreateBuilder();
         builder.Configuration.AddInMemoryCollection(appSettings);
-        builder.Configuration.AddPemFiles("instance.crt", "instance.key");
+        builder.Configuration.AddCertificate("Eureka", "instance.crt", "instance.key");
```

Comment:
`AddCertificate` only puts the files into `IConfiguration`, without setting up change tracking. I think that `AddEurekaDiscoveryClient` internally should call `services.ConfigureCertificateOptions()` (assuming that doesn't log errors or throw when the certificate path is not in configuration). And expect users to add the certificate paths to `appsettings.json` instead of calling this method, so that the paths can change at runtime.

Once this is merged, we should update https://github.com/SteeltoeOSS/Documentation/blob/v4/api/v4/discovery/netflix-eureka.md#configuring-mutual-tls accordingly.